src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -33,93 +43,53 @@ import {
   selectCustomRepositories,
   selectDistribution,
   addPackage,
+  addRecommendedRepository,
+  removeRecommendedRepository,
 } from '../../../../store/wizardSlice';
+import { useGetEnvironment } from '../../../../Utilities/useGetEnvironment';
 
 export type IBPackageWithRepositoryInfo = {
   name: Package['name'];
   summary: Package['summary'];
   repository: string;
 };
 
-const EmptySearch = () => {
-  return (
-    <Tr>
-      <Td colSpan={4}>
-        <Bullseye>
-          <EmptyState variant={EmptyStateVariant.sm}>
-            <EmptyStateHeader icon={<EmptyStateIcon icon={SearchIcon} />} />
-            <EmptyStateBody>
-              Search above to add additional
-              <br />
-              packages to your image
-            </EmptyStateBody>
-          </EmptyState>
-        </Bullseye>
-      </Td>
-    </Tr>
-  );
-};
-
-const NoResultsFound = () => {
-  return (
-    <Tr>
-      <Td colSpan={4}>
-        <Bullseye>
-          <EmptyState variant={EmptyStateVariant.sm}>
-            <EmptyStateHeader titleText="No results found" headingLevel="h4" />
-            <EmptyStateBody>Adjust your search and try again</EmptyStateBody>
-          </EmptyState>
-        </Bullseye>
-      </Td>
-    </Tr>
-  );
-};
-
-const TooManyResults = () => {
-  return (
-    <Tr>
-      <Td colSpan={4}>
-        <Bullseye>
-          <EmptyState variant={EmptyStateVariant.sm}>
-            <EmptyStateHeader
-              titleText="Too many results to display"
-              headingLevel="h4"
-            />
-            <EmptyStateBody>
-              Please make the search more specific and try again
-            </EmptyStateBody>
-          </EmptyState>
-        </Bullseye>
-      </Td>
-    </Tr>
-  );
-};
-
 const Packages = () => {
   const dispatch = useDispatch();
 
   const arch = useAppSelector((state) => selectArchitecture(state));
   const distribution = useAppSelector((state) => selectDistribution(state));
+
   const customRepositories = useAppSelector((state) =>
     selectCustomRepositories(state)
   );
+
+  const { data: epelRepo, isSuccess: isSuccessEpelRepo } =
+    useGetRepositoryQuery({
+      uuid:
+        distribution === 'rhel-8'
+          ? '3012f536-eb7c-4cfc-838d-a0cfd553224c'
```

Comment:
This feels like a "magic uuid"... how do we know the appropriate UUIDs?

Reply:
Those are queried from the content sources API and also seem to be the reason why the search doesn't work on popular repos that are not added - non-added repos don't have any uuid nor existing_name (although the suggested_name seems to be working for search).

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/996c22dc-bdbf-4cf8-a528-454dbcf0b5b2)

I wonder if `searchRpm` goes through the added repos only then. I'm not getting any results with popular non-added repo, but when it's added the results are there even though the url of the repo remains the same.

Reply:
Alright, so I was able to get rid of the uuids which indeed felt a bit like magic values. The repo is now fetched based on url which is much easier to confirm in content sources.